src/mlpack/bindings/R/CMakeLists.txt

```
@@ -230,7 +230,7 @@ if (BUILD_R_BINDINGS)
   install(CODE
       "execute_process(
          COMMAND R CMD INSTALL mlpack_${PACKAGE_VERSION}.tar.gz
```

```suggestion
         COMMAND ${R_EXECUTABLE} CMD INSTALL mlpack_${PACKAGE_VERSION}.tar.gz
```
Thanks @rcurtin for looking into this. I guess this will be more appropriate here, Let me know what you think?

---

Ah, agreed, thanks!